A6-client-retries.md

```diff
@@ -405,11 +405,11 @@ The retry policy is transmitted to the client through the service config mechani
         "maxAttempts": number,
 
         // Exponential backoff parameters. The initial retry attempt will occur at
-        // random(0, initialBackoffMs). In general, the nth attempt will occur at
+        // random(0, initialBackoff). In general, the nth attempt will occur at
         // random(0,
-        //   min(initialBackoffMs*backoffMultiplier**(n-1), maxBackoffMs)).
-        "initialBackoffMs": number,  // Required. Must be greater than zero.
-        "maxBackoffMs": number,  // Required. Must be greater than zero.
+        //   min(initialBackoff*backoffMultiplier**(n-1), maxBackoff)).
+        "initialBackoff": string,  // Required. Long decimal with "s" append
```

Comment:
Might want to explicitly note that these fields will be the JSON form of the protobuf Duration message as described in https://developers.google.com/protocol-buffers/docs/proto3#json.